build.sh

```diff
@@ -34,8 +47,17 @@ cmake --build . -- -j $(nproc)
 
 # If we are asked to install, we do this here
 # Otherwise, we simply copy the binary one level up
-if [[ "${1}" == "install" || "${2}" == "install" ]]; then
-    sudo make install
+if [[ -n "${install}" ]]; then
+    echo "Installing pihole-FTL"
+    SUDO=$(which sudo)
+    ${SUDO} make install
 else
+    echo "Copying compiled pihole-FTL binary to repository root"
     cp pihole-FTL ../
 fi
+
+if [[ -n "${test}" ]]; then
+    cd ..
+    ./test/run.sh
+    cd cmake
```

Comment:
Why this final `cd`?

Reply:
```suggestion
```

Typo again, initially I had the test section before the install/copy section